package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "identity-reporting",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "test": "run-p test:*",
+    "test:typecheck": "tsc",
+    "test:lint": "eslint src --ext .js",
```

`.js` is default, so I think it can be dropped.

```suggestion
    "test:lint": "eslint src",
```